lib/std/secrets.nim

```diff
@@ -4,7 +4,7 @@
 ## | :---         | ----:       |
 ## | Windows| `BCryptGenRandom`_ |
 ## | Linux| `getrandom`_ system call when available, otherwise `/dev/urandom`_ will be used|
-## | MacOS| `getentropy`_ system call when available, otherwise `/dev/urandom`_ will be used|
+## | MacOSX| `getentropy`_ system call when available, otherwise `/dev/urandom`_ will be used|
```

Comment:
> `otherwise `/dev/urandom`_ will be used|`  ?

where in the code does it do this fallback for osx? (and is that fallback needed there?)

Reply:
It haven't been implemented.

Reply:
Does Nim provide some convenient methods to check whether a C flags is defined or not>
https://android.googlesource.com/platform/external/expat/+/refs/heads/master/expat_config.h#31

Reply:
proposal 5 from https://github.com/nim-lang/RFCs/issues/205 would allow doing this in a robust, clean way; until then we need hacks similar to `NIM_EmulateOverflowChecks`